Automattic/newspack-theme
# Update how the primary content width is handled

### All Submissions:


### Changes proposed in this Pull Request:

This is some additional reworking of the theme's widths. It shouldn't actually _look_ different, but it should make adding a sidebar easier than the original styles. Instead of setting widths on individual items, like the post title, content, meta and previous/next navigation, the width has been moved to the containing element. 

It also has the added benefit of removing a bunch of CSS (that I'm sure will be re-added with the sidebar!). 

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Not to oversimplify things, but give things a quick once-over and make sure they don't really look any different :) 

### Other information:
